test/aws/scaleup.yml

```diff
@@ -27,10 +27,7 @@
       group: root
       mode: 0644
   - name: Initialize openshift repos
-    import_role:
-      name: openshift_repos
-    vars:
-      openshift_version: "4.0"
+    import_tasks: centos_repos.yml
```

Comment:
Go ahead and change this to `additional_repos.yml` since we are not adding centos repos any longer.